app/helpers/ugc_form_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Helper module for common html atributes or other repetitve code
+# that is used when generating the ugc form(s)
+module UGCFormHelper
```

Comment:
If we are always going to use these options on date inputs, we could probably DRY up the form view even more by creating our own DateInput, redefining and inheriting from the Simple Form one, per https://github.com/plataformatec/simple_form#custom-inputs, and setting the default values for input_html and html5.